src/thrift/patch/0002-cve-2017-1000487.patch

```diff
@@ -0,0 +1,11 @@
+--- ./thrift-0.11.0/contrib/thrift-maven-plugin/pom.xml	2021-03-29 05:20:22.046039640 +0000
++++ ./thrift-0.11.0/contrib/thrift-maven-plugin/pom_new.xml	2021-03-29 05:48:30.725383841 +0000
```

Comment:
Is this a patch from upstream? If not, why did you change filename?

Reply:
@qiluo-msft , can we simply change ./thrift-0.11.0/contrib/thrift-maven-plugin/pom_new.xml to ./thrift-0.11.0/contrib/thrift-maven-plugin/pom.xml.
I tested src/thrift/patch/0002-cve-2017-1000487.patch, there is no difference, when applying the patch by below command:
patch -p1 < ../patch/0002-cve-2017-1000487.patch